satchless/payment/__init__.py

```diff
@@ -4,6 +4,14 @@ def __init__(self, error_message):
         self.error_message = error_message
 
 
+class PaymentsGatewayReceiptFormError(PaymentFailure):
+    def __init__(self, form):
+        error_message = ', '.join(
```

Comment:
I believe you are thinking of `form._errors`, which is distinct from form.errors.